java-client/session/src/main/java/io/deephaven/client/impl/BatchTableRequestBuilder.java

```diff
@@ -390,15 +390,17 @@ public void visit(AsOfJoinTable aj) {
         }
 
         private static AsOfJoinTablesRequest.MatchRule adapt(AsOfJoinRule rule) {
+            // todo: fix gRPC?
```

Comment:
We'll update gRPC in another PR before this goes into a release.

Reply:
yep